foundations/foundation-config/src/main/java/org/apache/servicecomb/config/archaius/sources/MicroserviceConfigLoader.java

```diff
@@ -28,7 +28,9 @@
 public class MicroserviceConfigLoader extends YAMLConfigLoader {
   private static final Logger LOGGER = LoggerFactory.getLogger(MicroserviceConfigLoader.class);
 
-  private static final String ADDITIONAL_CONFIG_URL = "cse.configurationSource.additionalUrls";
+  private static final String ADDITIONAL_CONFIG_URL = "servicecomb.configurationSource.additionalUrls";
```

Comment:
not compatible?
because we only copy config items in configuration source, not in system.property?

Reply:
Same as above

Reply:
i remember someone already used this feature

Reply:
Throw an error for this wrong configuration and force users change to correct key.

Reply:
I deleted the README file and add the contents to java-chassis-doc, please see that PR.